packages/eslint-config-udemy-website/index.js

```diff
@@ -21,9 +20,9 @@ module.exports = {
         },
     },
     rules: {
-        'filenames/match-regex': ['error', '^(?:[a-z0-9\\-]+(?:\\.(?:jqui-widget|ng-(?:constant|controller|directive|factory|filter|provider|service)|react-(?:component|proptypes)|mobx-(?:model|store)))?(?:\\.spec)?)$'],
-        'lodash/import-scope': ['error', 'method'],
+        'filenames/match-regex': ['error', '^(?:[a-z0-9\\-]+(?:\\.(?:ng-(?:constant|controller|directive|factory|filter|provider|service)|react-(?:component|proptypes)|mobx-(?:model|store)))?(?:\\.spec)?)$'],
```

Comment:
https://trello.com/c/tKqRRBdE/1911-after-jqui-is-deleted-remove-jqui-widget-from-eslint-config-udemy-website-filenames-match-regex